mne/forward/_make_forward.py

```diff
@@ -444,8 +511,14 @@ def make_forward_solution(info, trans, src, bem, fname=None, meg=True,
                 command_line=cmd, bads=info['bads'])
     logger.info('')
 
-    megcoils, compcoils, eegels, megnames, eegnames, meg_info = \
-        _prep_channels(info, meg, eeg, ignore_ref)
+    megcoils, compcoils, megnames, meg_info = [], [], [], []
+    eegels, eegnames = [], []
+
+    if meg:
+        megcoils, compcoils, megnames, meg_info = \
+            _prep_meg_channels(info, ignore_ref=ignore_ref, verbose=verbose)
+    if eeg:
+        eegels, eegnames = _prep_eeg_channels(info, verbose=verbose)
```

Comment:
Same here, this would have thrown an error before if `not (meg or eeg)`, now I don't really know what it will do...?